### PR TITLE
[de] Add Rollo to cover shutter and Vorhang to abdeckungen

### PR DESCRIPTION
**Pull request by jhbruhn**

### CODE REVIEW

**Review by mib1185 — CHANGES_REQUESTED** [review]
Hi @jhbruhn 
please cover these additions with test sentences, thx

**Comment by mib1185** [comment]
there are failing tests, please fix them

**Comment by jhbruhn** [comment]
Cloud this PR get another review?

**Review by steffenrapp — APPROVED** [review]
LGTM 👍

File: tests/de/cover_HassTurnOff.yaml [hunk 1]
```diff
@@ -14,6 +14,16 @@ tests:
         name: Rollo links
         domain: cover
     response: geschlossen
+  - sentences:
+      - schließ den Vorhang links
+      - Vorhang links schliessen
+      - mach den Vorhang links herunter
```
Comment (mib1185) [inline tests/de/cover_HassTurnOff.yaml:20]: tbh, this sounds weird (_this counts also for all similar others_)

**Review by mib1185 — APPROVED** [review]
HI @jhbruhn 
thanks for this contribution, overall it looks good 👍 
Just one comment, but it is just a comment 😃

File: tests/de/cover_HassTurnOff.yaml (hunk 1 above)
Comment (jhbruhn) [reply tests/de/cover_HassTurnOff.yaml:20]: I agree, I just did not want to make the RegEx too complicated - better some more cases than too little cases was my line of thought here. Anyways, thanks for merging!

**Pull request merged**